docs/architecture.md

```diff
@@ -21,52 +21,52 @@ This leads to constant development and DevOps/MLOps work.
 The fact that you probably need to use different types of storage (such as files and AWS S3 buckets) and various databases, further complicates the implementation.
 
-Once you finished development you want to serve models online or build real-time pipelines without having to refactor or 
+Once the development is complete, it's time to serve models online or build real-time pipelines without having to refactor or 
 re-implement the logic, or call an army of developers to help.
 
 Wouldn't it be great if you could write the code once, using your preferred development environment and simple "local" semantics, and then run it as-is on different platforms?
-Imagine a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
+Imagine having a layer that automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, deployment to production, monitoring, and more.
 A world of easily developed, published, or consumed data or ML "functions" that can be used to form complex and large-scale offline or real-time ML pipelines.
 
-In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code reuse across projects and companies and thus further accelerate your work.
+In addition, imagine a marketplace of ML functions that includes both open-source templates and your internally developed functions, to support code for reuse across projects and companies and thus further accelerate your work.
 
-<b>This is the goal of MLRun, Simplify & Accelerate Time To Production.</b>
+<b>This is the goal of MLRun - Simplify & Accelerate Time To Production.</b>
 
 ## Architecture 
 
 <img src="_static/images/mlrun-architecture.png" alt="mlrun-architecture" width="800"/>
 
-MLRun comprises of the following layers:
+MLRun comprises the following layers:
 * **Feature & Artifact Store** - Handle the ingestion, processing, metadata and storage of data and features across multiple repositories and technologies
 * **Elastic Serverless Runtimes** - Convert simple code to scalable and managed micro-services with 
-  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  .
+  workload specific runtime engines (Kubernetes jobs, Nuclio, Dask, Spark, Horovod, etc.)  
 * **ML Pipeline Automation** - Automated data preparation, model training & testing, 
-  deployment of production (real-time) pipelines, and end to end monitoring.
+  deployment of production (real-time) pipelines, and end-to-end monitoring
 * **Central Management** - Unified portal, UI, CLI, and SDK to manage the entire MLOps workflow which is 
-  accessible from everywhere. 
+  accessible everywhere
```

Comment:
This section should be aligned with updates done to the similar architecture documentation in **docs/index.rst** (and the root **README.md** file:
```markdown
MLRun is composed of the following layers:

- **Feature and Artifact Store** &mdash;
    handles the ingestion, processing, metadata, and storage of data and features across multiple repositories and technologies.
- **Elastic Serverless Runtimes** &mdash;
    converts simple code to scalable and managed microservices with workload-specific runtime engines (such as Kubernetes jobs, Nuclio, Dask, Spark, and Horovod).
- **ML Pipeline Automation** &mdash;
    automates data preparation, model training and testing, deployment of real-time production pipelines, and end-to-end monitoring.
- **Central Management** &mdash;
    provides a unified portal for managing the entire MLOps workflow.
    The portal includes a UI, a CLI, and an SDK, which are accessible from anywhere.
```